src/components/Storage/Canvas/DeleteAllButton/DeleteAllButton.test.tsx

```diff
@@ -27,7 +28,7 @@ describe('DeleteAllButton', () => {
   // TODO: investigate emulators + jsdom
   it.skip('deletes all files', async () => {
```

Comment:
It seems like this test wont be run due to the `it.skip` test rule here

Reply:
I put a TODO to investigate these soon